src/ble/BleLayer.cpp

```diff
@@ -308,6 +308,15 @@ void BleLayer::Shutdown()
 
 void BleLayer::CloseAllBleConnections()
 {
+    //Cancel any ongoing attempt to establish new BLE connection
```

Comment:
Is there a reason to not just call `CancelBleIncompleteConnection()` here?

Reply:
That works too. I'll change it to call CancelBleIncompleteConnection().